rpc/wrtc_base_channel.go

```diff
@@ -144,11 +144,11 @@ func (ch *webrtcBaseChannel) closeWithReason(err error) error {
 	ch.closedReason = err
 	ch.cancel()
 	ch.bufferWriteCond.Broadcast()
+	ch.activeBackgroundWorkers.Wait()
```

Comment:
we may as well wait after we cancel

Reply:
This is actually probably wrong. Since a read and write can't be context guarded really in go, we need to rely on the close to kill the operation. Will see if I can work around the errors resulting in the race here